recipe/meta.yaml

```
@@ -1,24 +1,27 @@
 {% set name = "debugpy" %}
-{% set version = "1.8.11" %}
+{% set version = "1.8.16" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 6ad2688b69235c43b020e04fecccdf6a96c8943ca9c2fb340b8adc103c655e57
+  sha256: 31e69a1feb1cf6b51efbed3f6c9b0ef03bc46ff050679c4be7ea6d2e23540870
 
 build:
   number: 0
   entry_points:
     - debugpy = debugpy.server.cli:main
+  ignore_run_exports:
+    - libcxx  # [osx]
 
 requirements:
   build:
     - python
-    - {{ compiler('c') }}
-    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}           # [unix]
+    - {{ compiler('cxx') }}         # [unix]
+    - pkg-config                    # [linux]
```

```suggestion
    - {{ stdlib('c') }}
    - { compiler('c') }}
    - {{ compiler('cxx') }}
    - pkg-config
```
It's using compilers on Windows, need to include them in the recipe so that they get configured correctly.